iothub/service/src/Messaging/DeliveryAcknowledgement.cs

```diff
@@ -1,11 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// Specifies the different acknowledgement levels for message delivery.
     /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
Was this class broken without this JSON converter? Or is this just to make explicit what was previously implicit?